### ISSUE TITLE
Title in tool bar is not updated on tab change.

### ISSUE THREAD

**Issue by gety9** [plateaukao/browser#70]
Very frequently (but seems not always) Eink bro enters state when title is not updated on tab change. 
(it's not related to eink refresh, i tried to "full page refresh")

Example:
1, Last opened new tab is best buy and has correct title.
![Screenshot_20220103-141303](https://user-images.githubusercontent.com/16415882/147977059-899434aa-84d4-4844-887c-5d379c62ff60.png)

2, Change to another tab (ebay), but title does not change to ebay.
![Screenshot_20220103-141331](https://user-images.githubusercontent.com/16415882/147977490-b8375116-73e8-4c24-b6d3-f5cf67c50581.png)

3, Change to another tab (google), same.
![Screenshot_20220103-141347](https://user-images.githubusercontent.com/16415882/147977569-5b38e11a-b996-4344-9708-c34c586968fb.png)

4, Open new tab, but don't enter anything (press "v" on right) Now has no title
Sorry forget, screenshot :)

5, Visit another tab, title is not updated. (stays same as in step 4)
![Screenshot_20220103-141436](https://user-images.githubusercontent.com/16415882/147978118-31f838b6-96a9-43ad-9515-6d5c653a4258.png)

6, Visit another tab, no update (title is from 4)
![Screenshot_20220103-141503](https://user-images.githubusercontent.com/16415882/147978188-850f6a7f-a80f-4e22-851a-50ca164886b7.png)

7, Alt+tabing to another app and back seem to force title refresh
![Screenshot_20220103-141537](https://user-images.githubusercontent.com/16415882/147978588-b8b8a2d2-c0ae-4259-b56f-85a1a5c3d25c.png)

Basically in this state title is updated on
- new tab
- return to app (alt tab)

but not tab change.

Titles on tab preview window (that were fixed in https://github.com/plateaukao/browser/issues/68) are always correct, it's buttom left in tool bar Title.

**Comment by plateaukao** [comment]
would be fixed in next release.

**Comment by plateaukao** [comment]
fixed in https://github.com/plateaukao/browser/releases/tag/v8.26.0
